lib/reporting/identity_verification_report.rb

```diff
@@ -136,75 +158,77 @@ def merge(other)
       )
     end
 
-    def idv_final_resolution
-      data[Events::IDV_FINAL_RESOLUTION].count
+    def idv_final_resolution(data_set = data['all'])
+      data_set[Events::IDV_FINAL_RESOLUTION].count
     end
 
-    def idv_final_resolution_verified
-      data[Results::IDV_FINAL_RESOLUTION_VERIFIED].count
+    def idv_final_resolution_verified(data_set = data['all'])
+      data_set[Results::IDV_FINAL_RESOLUTION_VERIFIED].count
     end
 
-    def idv_final_resolution_gpo
-      data[Results::IDV_FINAL_RESOLUTION_GPO].count
+    def idv_final_resolution_gpo(data_set = data['all'])
+      data_set[Results::IDV_FINAL_RESOLUTION_GPO].count
     end
```

Comment:
I think we might be better off splitting some of these property methods into another object.

So one object creates the CW query and loads the rows, then we instantiate N "result/data" objects that can query their own "data" hash, filtered by A/B test (or all, etc)

Reply:
Agreed. This is a spike so I'll create a ticket to do this as follow-up.

Reply:
I took a stab at making a separate `Result` object here: https://github.com/18F/identity-idp/compare/dprice-lg-11661-ab-tests-in-reports...margolis-ab-test-report-v2